pkg/util/jib/configuration.go

```diff
@@ -50,8 +53,8 @@ type JibProfile struct {
 }
 
 // Create a Configmap containing the default jib profile.
-func CreateProfileConfigmap(ctx context.Context, c client.Client, kit *v1.IntegrationKit) error {
-	profile, err := jibMavenProfile()
+func CreateProfileConfigmap(ctx context.Context, c client.Client, catalog *camel.RuntimeCatalog, kit *v1.IntegrationKit) error {
```

Comment:
In order to reduce the coupling, I'd instead create the profile externally and pass it in this func instead of the catalog.